apps/displays/OSVR_HDK_1_3.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "OSVR",
+      "model": "HDK",
+      "num_displays": 1,
+      "Version": "1.2",
+      "Note": "Suitable for HDK 1.0-1.2"
+    },
+    "field_of_view": {
+      "monocular_horizontal": 90,
+      "monocular_vertical": 96.73,
+      "overlap_percent": 100,
+      "pitch_tilt": 0
+    },
+    "resolutions": [
+      {
+        "width": 1920,
+        "height": 1080,
+        "video_inputs": 1,
+        "display_mode": "horz_side_by_side",
+        "swap_eyes": 0
+      }
+    ],
+    "distortion": {
+        "distance_scale_x": 1,
+        "distance_scale_y": 1,
+        "polynomial_coeffs_red": [ 0, 1, -0.95, 1.88, 1.62, 1.11 ],
+        "polynomial_coeffs_green": [ 0, 1, -0.95, 1.88, 1.62, 1.11 ],
+        "polynomial_coeffs_blue": [ 0, 1, -0.95, 1.88, 1.62, 1.11 ]
```

Comment:
They are parsed by RenderManager as is.  Was using the same construction as before, which had K's for red,green,blue.  Not wedded to any particular format.  Same could be said of the distance_scale, which could have been a vector but I thought having X and Y explicit made sense.  Could nest X and Y of course.

Just need to know what it changes to if it changes so can adjust the parser.  Would like this all to be parsed in Core, of course, and sent over in a structure.  Jeremy wanted to get things running sooner, so we're in a middle state.

Note that in the immediate future, we're going to need to extend this to pass a (large) mesh, so we may want to hold off on pulling into core until that happens.